test-cases/scylla-operator/longevity-scylla-operator-3h-multitenant.yaml

```diff
@@ -30,4 +30,6 @@ nemesis_seed: ['385', '543']
 space_node_threshold: [64424, 64423]
 
 user_prefix: 'longevity-scylla-operator-3h-multitenant'
-cluster_health_check: false
+cluster_health_check: true
+
+k8s_enable_tls: true
```

Comment:
Do we want to have only this variant?
Maybe we want to create second job and then just choose what we want to run?

Reply:
Yes we want only this variant, we are targeting for what serverless is, and it would be only with it.

in case we need to compare we can rerun without it